File: CJY/BJ/No_2869.java
import java.io.*;

public class No_2869 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] arr = br.readLine().split(" ");
        br.close();

        int a = Integer.parseInt(arr[0]);
        int b = Integer.parseInt(arr[1]);
        int v = Integer.parseInt(arr[2]) - a;
        int p = 0;
        int c = 1;

        while (true) {
            p += a;
            if(p >= (v  % (a - b)) + a) break;
            p -= b;
            c++;
        }

        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        bw.write(String.valueOf((v / (a - b)) + c));
        bw.flush();
        bw.close();
    }
}

File: CJY/BJ/No_2869.js
const rl = require("readline").createInterface({input: process.stdin, output: process.stdout});

rl.on("line", (s) => {
    const arr = s.split(" ");
    let p = 0, c = 1;
    let a = parseInt(arr[0]);
    let b = parseInt(arr[1]);
    let v = parseInt(arr[2]) - a;

    while (true) {
        p += a;
        if(p >= (v  % (a - b)) + a) break;
        p -= b;
        c++;
    }
    console.log(~~(v / (a - b)) + c);
    rl.close();
}).on("close", () => {process.exit();});